report/renderer/category-renderer.js

```diff
@@ -571,7 +572,11 @@ export class CategoryRenderer {
       }
 
       const description = clumpId === 'manual' ? category.manualDescription : undefined;
-      const clumpElem = this.renderClump(clumpId, {auditRefs, description});
+      const scoreOutOf100 = Math.round(Number(category.score) * 100);
+      // Expand the manual a11y checks upon scoring 100.
+      const openByDefault =
+        (category.id === 'accessibility' && clumpId === 'manual' && scoreOutOf100 === 100);
```

Comment:
I'm open to expanding the manual section for all categories not just a11y. WDYT?

Reply:
It could make sense to always expand the manual section (i.e. `{openByDefault: true}`).

Reply:
SGTM, added `{openByDefault: true}` for PWA.